content/collections/tags/form-create.md

```diff
@@ -55,7 +55,7 @@ variables:
     name: error
     type: array
     description: |
-      An array of validation errors indexed by **field name**. For example: `{{ error:email }}`
+      An array of validation errors indexed by **field name**. For example: `{{ error:email }}`, to access the error with a variable use it like an object `{{ error[variable_name] }}`
```

Comment:
Not sure if that hint is needed since it's just how dynamic access for arrays in Antlers works (https://statamic.dev/antlers#dynamic-access)

Reply:
makes sense, just saying from a newcomers perspective.